### PR TITLE
Feat: error handlers components

### PR DESCRIPTION
**Pull request by TheMadNerd**
#36

### CODE REVIEW

File: react-app/src/pages/popup/pages/error/ErrorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+);
+
+export const ErrorPage = () => {
+  return (
+    <>
+      <GlobalStyles />
+      <StyledHomePageWrapper>
+        <StyledErrorContainer>
+          <StyledIcon src={octagonError} alt={'Boundary error icon'} width={60} height={60} />
+          <div>
```
Comment (masterHAWK99) [inline react-app/src/pages/popup/pages/error/ErrorPage.tsx:37]:
```suggestion
          <p>
```

File: react-app/src/pages/popup/pages/error/ErrorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            BŁĄD !
+            <p style={{ fontSize: '16px' }}>
```
Comment (masterHAWK99) [inline react-app/src/pages/popup/pages/error/ErrorPage.tsx:39]:
```suggestion
            <span style={{ fontSize: '16px' }}>
```

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

File: react-app/src/pages/popup/pages/error/ErrorPage.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          <StyledIcon
+            style={{ marginBottom: '30px' }}
+            src={octagonError}
+            alt={'Boundary error icon'}
+            width={60}
+            height={60}
+          />
+          BŁĄD !
+          <span style={{ fontSize: '16px' }}>
+            Wystąpił błąd krytyczny, odśwież stronę, gdy to nie pomoże, skontaktuj się z developerem
```
Comment (PatrykKuniczak) [inline react-app/src/pages/popup/pages/error/ErrorPage.tsx:44]:
```suggestion
            Wystąpił błąd krytyczny, uruchom ponownie wtyczkę, gdy to nie pomoże, skontaktuj się z developerem
```

**Review by PatrykKuniczak — APPROVED** [review]
LGTM

**Review by masterHAWK99 — APPROVED** [review]

File: react-app/public/locales/en/translation.json [hunk 4]
```diff
@@ -49,5 +49,8 @@
   "successfulLogout": "Successfully logged out",
   "successfulDeleteAccount": "Successfully deleted account",
   "videosLoaded": "From under provided keywords, {{loadedVideosAmount}} videos were fetched, check them out in extension",
-  "fallbackError": "Unexpected error occurred, please contact with developer"
+  "fallbackError": {
```
Comment (PatrykKuniczak) [inline react-app/public/locales/en/translation.json:52]: This message is used as fallback to toasts on useQuery, you can't overwrite it easly, this change, break the app, let's check reference before make changes in the future, for this let's create heading as seprate prop

**Review by PatrykKuniczak — CHANGES_REQUESTED** [review]

**Review by masterHAWK99 — APPROVED** [review]

File: react-app/public/locales/pl/translation.json [hunk 5]
```diff
@@ -46,6 +46,7 @@
   "successfulLogout": "Pomyślnie wylogowano",
   "successfulDeleteAccount": "Pomyślnie usunięto konto",
   "videosLoaded": "Znaleziono {{loadedVideosAmount}} wideo podczas przeszukiwania podanych słów kluczowych. Proszę sprawdzić we wtyczce.",
-  "fallbackError": "Wystąpił niespodziewany błąd, skontaktuj sie z developerem",
+  "fallbackError": "Wystąpił niespodziewany błąd, skontaktuj sie z developerem!",
+  "heading": "BŁĄD",
```
Comment (PatrykKuniczak) [inline react-app/public/locales/pl/translation.json:50]:
```suggestion
  "fatalErrorHeading": "BŁĄD",
```

cause IMO, it's more meaningful, if you have many props, then this 'heading' could be everything or nothing :)

**Review by PatrykKuniczak — CHANGES_REQUESTED** [review]

**Review by PatrykKuniczak — APPROVED** [review]
LGTM

**Pull request merged**